### PR TITLE
EDSC-1054 Add error message when un-supported file is uploaded.

### PR DESCRIPTION
**Pull request by mightynimble**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -299,6 +299,3 @@ DEPENDENCIES
   unicorn
   vcr
   whenever
-
-BUNDLED WITH
```
Comment (bilts) [inline Gemfile.lock]: What happened here?

**Comment by bilts** [comment]
This is good, but I think we should additionally have some one-off instructions specifically for .shp.

File: Gemfile.lock (hunk 1 above)
Comment (mightynimble) [inline Gemfile.lock]: After v1.10, bundler adds it automatically. I was using an older version which removes it on ```bundle install```

Comment (bilts) [inline Gemfile.lock]: No, I was just curious.

**Comment by bilts** [comment]
When I hover to remove the file, the error text shows up in both the tooltip and inline. Can we make it so the error text only shows up inline?

**Pull request merged**